LetSwift/View/Events/Views/EventsView.swift

```diff
@@ -11,12 +11,17 @@ import SwiftUI
 struct EventsView: View {
     
     // MARK: - Properties
-    @State private var eventCellStates: [Event] = []
     @Binding var payload: String
+    private var viewModel = EventsViewModel()
```

Comment:
```suggestion
    private let viewModel = EventsViewModel()
```

`viewModel`이 런타임에서 굳이 변경되지 않은 값이면, `let`으로 선언하는 것은 어떠신가요!

Reply:
> `viewModel`이 런타임에서 굳이 변경되지 않은 값이면, `let`으로 선언하는 것은 어떠신가요!

311f359 
State를 제거하다가 놓쳤네요..! 꼼꼼한 확인 감사합니다!